docs/DeveloperGuide.md

```diff
@@ -262,64 +305,42 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* freelance software developers
-* value privacy 
-* self-hosting
+* has a need to manage a significant number of contacts
+* prefer desktop apps over other types
+* can type fast
+* prefers typing to mouse interactions
+* is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app, along with privacy features.
+**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
 
 
 ### User stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-### User stories
+| Priority | As a …​                                    | I want to …​                 | So that I can…​                                                        |
```

Comment:
If you're updating the user stories, it might be good to add the other user stories as written in the excel spreadsheet.